Blockzilla/SettingsViewController.swift

```diff
@@ -390,9 +338,25 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         
         return cell
     }
+    
+    func numberOfRows(for section: Section) -> Int {
+        switch section {
+        case .general: return 1
+        case .privacy:
+            if authenticationManager.canEvaluatePolicy { return 3 }
+            return 2
+        case .usageData: return 1
+        case .studies: return 1
+        case .search: return 3
+        case .siri: return 3
+        case .integration: return 1
+        case .mozilla: return 3
+        case .secret: return 1
+        }
+    }
 
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        (sections[section] == .mozilla && onboardingEventsHandler.shouldShowNewOnboarding()) ? 2 : sections[section].numberOfRows
+        (sections[section] == .mozilla && onboardingEventsHandler.shouldShowNewOnboarding()) ? 2 : numberOfRows(for: sections[section])
     }
```

Comment:
This is implementation is fine here, but at cellFor row there is an issue for .mozilla case. I think that the code for index 0 somehow has been lost.
This is the current implementation :
```

 case .mozilla:
            if indexPath.row == 1 {
                aboutClicked()
            } else if indexPath.row == 2 {
                let appId = AppInfo.config.appId
                if let reviewURL = URL(string: "https://itunes.apple.com/app/id\(appId)?action=write-review"
...
```
At the moment AboutFirefox is not working when tapping the row and it appears when Rate the app is selected.
The case for indexPath.row == 0 should be added and About Firefox should be the first cell in section if the new onboarding is enabled. Also Rate app should be the second or the third row, if old onboarding is enabled.

Reply:
Issue also reproduces on main. Fixed in this issue.